src/main/java/uk/gov/hmcts/reform/pip/data/management/controllers/PublicationController.java

```diff
@@ -371,4 +371,14 @@ private void logManualUpload(String issuerId, String artefactId) {
             log.info(writeLog(issuerId, UserActions.UPLOAD, artefactId));
         }
     }
+
+
+    @ApiResponses({
+        @ApiResponse(code = 202, message = "Data Management - MI Data request accepted.")
+    })
+    @ApiOperation("Return the table of MI data")
+    @GetMapping("/mi-data")
+    public ResponseEntity<String> getMiData() {
+        return ResponseEntity.ok(publicationService.getMiData());
```

Comment:
This is returning a ok response code but the description above says it is returning a 202 upon success. I think 200 is fine in this case

Reply:
I've just switched it to an Accepted - feels like it's more in line with the other repos?

Reply:
And now I've undone that :p

Reply:
Thank you :)